laser_slam_ros/src/laser_slam_worker.cpp

```diff
@@ -1,3 +1,4 @@
+#include "laser_slam/benchmarker.hpp"
```

Comment:
Put below the other include, in a new block (see: https://github.com/ethz-asl/segmatch/wiki/Contributing-to-SegMatch#header-include-guide).